### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Nick-NCSU**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -304,7 +304,40 @@ From your AWS Management Console, perform the following steps:
...
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "events:PutEvents"
+                ],
+                "Resource": [
+                    "arn:aws:events:us-east-1:612791702201:event-bus/jupiter-integration-aws"
```
Comment (Nick-NCSU) [inline README.md:320]:
```suggestion
                    "arn:aws:events:<TARGET AWS REGION>:<JUPITERONE ACCOUNT ID>:event-bus/jupiter-integration-aws"
```

File: README.md [hunk 2]
```diff
@@ -304,7 +304,40 @@ From your AWS Management Console, perform the following steps:
...
+                ],
+                "Resource": [
+                    "arn:aws:events:<TARGET AWS REGION>:<JUPITERONE ACCOUNT ID>:event-bus/jupiter-integration-aws"
+                ]
+            }
+        ]
+    }
+    ```
+    and a trust relationship which looks like:
+    ```
```
Comment (Nick-NCSU) [inline README.md:327]:
```suggestion
    ```json
```

File: README.md [hunk 3]
```diff
@@ -304,7 +304,40 @@ From your AWS Management Console, perform the following steps:
...
+                    "events:PutEvents"
+                ],
+                "Resource": [
+                    "arn:aws:events:<TARGET AWS REGION>:<JUPITERONE ACCOUNT ID>:event-bus/jupiter-integration-aws"
+                ]
+            }
+        ]
+    }
+    ```
+    and a trust relationship which looks like:
```
Comment (ndowmon) [inline README.md:326]: I'm surprised that this role needs a trust relationship in order to put events onto the bus

Comment (ndowmon) [reply README.md:326]: Oh, i guess less surprised - `sts:AssumeRole`

**Review by ndowmon — APPROVED** [review]

**Pull request merged**